### ISSUE TITLE
Automated Backups enabled by default

### ISSUE THREAD

**Issue by UH-Nerion** [solusio/solusiovps#14]
Automated backups are enabled by default, with no option to disable it.

**Comment by JuergenLeber** [comment]
The backups can be enabled/disabled in the product configuration.

**Comment by UH-Nerion** [comment]
I am talking about atuomated backups, not manual backups

**Comment by UH-Nerion** [comment]
In Product configuration I have "Enable Backups".
Does this mean enable automated backups? Then you have a false wording.

**Comment by JuergenLeber** [comment]
So you want "automated backups to be enabled by default with no option to disable it" or you meant that they are enabled but can't de disabled?

**Comment by UH-Nerion** [comment]
Yes, they are enabled by default with no option to disable it.

**Comment by JuergenLeber** [comment]
Okay, so currently if you enable backups in product configuration then they will be automatically done and in addition to that the customer can create manual backups. Do I get it right that you want to control that - meaning to additionally (besides enabling/disabling backups at all for a product) have a checkbox for "Automated backups" and "Enable manual backups"?

**Comment by UH-Nerion** [comment]
Yes seperated checkboxes for automated backups and manual backups.

**Comment by UH-Nerion** [comment]
PS: I think this can be fixed together with #6 issue.
Customer should be able to manage scheduled backups like in Solus.IO directly.

**Comment by JuergenLeber** [comment]
Yes, this can be combined!
